python/kserve/requirements.txt

```diff
@@ -20,8 +20,8 @@ avro>=1.10.1
 # boto3>=1.18.18
 # botocore>=1.21.18
 psutil>=5.0
-ray[serve]==1.5.0
-grpcio==1.38.1
+ray[serve]==1.8.0
+grpcio>=1.52.0
```

Comment:
Hey @Peilun-Li , as you also [updated](https://gitlab.zgtools.net/analytics/artificial-intelligence/ai-platform/sdks/internal/zillow-kserve/-/blob/zillow/kserve/python/kserve/requirements.txt#L23-25) these dependencies around the same time, can you please comment here? Do we want to pin to a specific higher version of ray or just relax the dependencies like you did? And does bumping up the minimum version of grpcio work for your llm changes?

Reply:
I guess my change can already be a superset of this PR as client side can always pin versions down more specifically? Or if Alex has any detailed requirements

Reply:
As per my understanding, the change Alex made were to make it installable locally on the M1 macbook (when used in aip-workflow-example). But I guess that's not super important, if Chris is unblocked. We can wait for the upgrade of the SDK to use the kserve client v0.10.2+.
@alexlatchford given Peilun and Chris are unblocked, can we close this MR?

Reply:
Yeah I think with the next steps you've articulated Ash let's keep the original plan to upgrade as normal, closing this MR as a result :)